pkgs/applications/emulators/ryujinx/default.nix

```diff
@@ -113,7 +113,7 @@ buildDotnetModule rec {
     install -D ../misc/Logo.svg $out/share/icons/hicolor/scalable/apps/Ryujinx.svg
 
     substituteInPlace $out/share/applications/Ryujinx.desktop \
-      --replace "Ryujinx %f" "$out/bin/Ryujinx %f"
+      --replace "Ryujinx.sh %f" "$out/bin/Ryujinx %f"
```

Comment:
(After installing Ryujinx.sh)
Is there any point in keeping the `substituteInPlace`?

Reply:
Hi. I'm not that good with nix.

 I think we still need this to give a full path to `Ryujinx.sh` in `Ryujinx.desktop`.

My issue is that the `Ryujinx.sh`, the script itself, picked `Ryujinx.Headless.SDL2` as an executable binary because it exists in the directory too. 
```shell
if [ -f "$SCRIPT_DIR/Ryujinx.Headless.SDL2" ]; then
    RYUJINX_BIN="Ryujinx.Headless.SDL2"
fi
```

Reply:
Uhm, I guess that a better script would be:
```
if [ -f "$SCRIPT_DIR/Ryujinx.Headless.SDL2" ]; then
    RYUJINX_BIN="Ryujinx.Headless.SDL2"
fi

if [ -f "$SCRIPT_DIR/Ryujinx.Ava" ]; then
    RYUJINX_BIN="Ryujinx.Ava"
fi

if [ -f "$SCRIPT_DIR/Ryujinx" ]; then
    RYUJINX_BIN="Ryujinx"
fi
```

Maybe that could be proposed upstream, so that we don't need to patch it ourselves?

Reply:
I made a proposal PR. See there.

Reply:
@SuperSamus I created separate launchers and wrappers to deal with this. Plz review.